src/main/java/com/bazaarvoice/maven/plugin/process/ProcessHealthCondition.java

```diff
@@ -68,6 +68,12 @@ private static int getResponseCode(URL url, SSLSocketFactory sslSocketFactory) {
 
             if (http instanceof HttpsURLConnection) {
                 ((HttpsURLConnection) http).setSSLSocketFactory(sslSocketFactory);
+                HttpsURLConnection.setDefaultHostnameVerifier(new HostnameVerifier() {
```

Comment:
We're checking to see if this can be a standalone instance instead of replacing the default globally